voila/app.py

```diff
@@ -348,6 +346,25 @@ def setup_template_dirs(self):
                 self.static_paths,
                 self.template_paths,
                 self.voila_configuration.template)
+            # look for possible template-related config files
+            template_conf_dir = [os.path.dirname(k) for k in self.nbconvert_template_paths]
```

Comment:
This will not give the proper path if k ends with a slash

Reply:
```suggestion
            template_conf_dir = [os.path.join(k, '..') for k in self.nbconvert_template_paths]
```

Reply:
Right, thank you! I definitely have to remove the check line 354 then.